salt/INSTALL.md

```diff
@@ -0,0 +1,32 @@
+# Install OpenSDS through Salt
+
+## Preparation
+
+Provision one or more GNU/Linux hosts so we can deploy OpenSDS.
+### Download installer 
+
+```
+sudo -s
+git clone https://github.com/<reponame>/opensds-installer.git /root/opensds-installer
+cd /root/opensds-installer
+git checkout dev-salt
+```
+
+### Site-specific data collection
+
+Review and update ``/root/opensds-installer/salt/site.j2`` parameters according to site requirements.
+
+### Install Salt
+```
+sudo /root/opensds-installer/salt/install.sh -i salt [ -m <SALT-MASTER-HOST> ]
+```
+i.e.
+```
+  sudo /root/opensds-installer/salt/install.sh -i salt [ -m 10.0.2.15 ]
+```
+
+### Install OpenSDS
+```
+sudo /root/opensds-installer/salt/install.sh -i opensds
```

Comment:
After I ran this command, some errors occur as below:
```shell
root@ubuntu:~/gopath/src/github.com/opensds/opensds-installer/salt# ./install.sh -i opensds
Running salt ...
local:
    ----------
    opensds:
        - default
root@ubuntu:~/gopath/src/github.com/opensds/opensds-installer/salt# [CRITICAL] Rendering SLS 'base:iscsi.target.install' failed: found character '%' that cannot start any token; line 85

---
[...]
    - watch:
      - file: iscsi_target_service_config
        
    - name: target

    - unless: %s %s    <======================



---
local:
    Data failed to compile:
----------
    Rendering SLS 'base:iscsi.target.install' failed: found character '%' that cannot start any token; line 85

---
[...]
    - watch:
      - file: iscsi_target_service_config
        
    - name: target

    - unless: %s %s    <======================
```

Is there something I missed?

Reply:
Hi @leonwanghui  thanks for looking at this.  I remember that problem.

Can you refresh your stack and error will go away.  
```
sudo -s
git clone https://github.com/noelmcloughlin/opensds-installer.git /root/opensds-installer   
cd /root/opensds-installer
git checkout dev-salt
sudo rm -fr /srv/formulas/* .  ## just in case
sudo /root/opensds-installer/salt/install.sh -i salt
```